Foundation/NSObjCRuntime.swift

```diff
@@ -255,75 +255,80 @@ internal let _NSClassesRenamedByObjCAPINotesInNetworking: [(swiftName: String, o
+        (XMLDocument.self, "NSXMLDocument"),
+        (XMLElement.self, "NSXMLElement"),
+        (XMLNode.self, "NSXMLNode"),
+        (XMLParser.self, "NSXMLParser"),
+        (Bundle.self, "NSBundle"),
+        (ByteCountFormatter.self, "NSByteCountFormatter"),
+        (Host.self, "NSHost"),
+        (DateFormatter.self, "NSDateFormatter"),
+        (DateIntervalFormatter.self, "NSDateIntervalFormatter"),
+        (EnergyFormatter.self, "NSEnergyFormatter"),
+        (FileHandle.self, "NSFileHandle"),
+        (FileManager.self, "NSFileManager"),
+        (Formatter.self, "NSFormatter"),
+        (InputStream.self, "NSInputStream"),
+        (ISO8601DateFormatter.self, "NSISO8601DateFormatter"),
+        (JSONSerialization.self, "NSJSONSerialization"),
+        (LengthFormatter.self, "NSLengthFormatter"),
+        (MassFormatter.self, "NSMassFormatter"),
+        (NotificationQueue.self, "NSNotificationQueue"),
+        (NumberFormatter.self, "NSNumberFormatter"),
+        (Operation.self, "NSOperation"),
+        (OperationQueue.self, "NSOperationQueue"),
+        (OutputStream.self, "NSOutputStream"),
+        (PersonNameComponentsFormatter.self, "NSPersonNameComponentsFormatter"),
+        (Pipe.self, "NSPipe"),
+        (Progress.self, "NSProgress"),
+        (PropertyListSerialization.self, "NSPropertyListSerialization"),
+        (RunLoop.self, "NSRunLoop"),
+        (Scanner.self, "NSScanner"),
+        (Stream.self, "NSStream"),
+        (Thread.self, "NSThread"),
+        (Timer.self, "NSTimer"),
+        (UserDefaults.self, "NSUserDefaults"),
+        (FileManager.DirectoryEnumerator.self, "NSDirectoryEnumerator"),
+        (Dimension.self, "NSDimension"),
+        (Unit.self, "NSUnit"),
+        (UnitAcceleration.self, "NSUnitAcceleration"),
+        (UnitAngle.self, "NSUnitAngle"),
+        (UnitArea.self, "NSUnitArea"),
+        (UnitConcentrationMass.self, "UnitConcentrationMass"),
+        (UnitConverter.self, "NSUnitConverter"),
+        (UnitConverterLinear.self, "NSUnitConverterLinear"),
+        (UnitDispersion.self, "NSUnitDispersion"),
+        (UnitDuration.self, "NSUnitDuration"),
+        (UnitElectricCharge.self, "NSUnitElectricCharge"),
+        (UnitElectricCurrent.self, "NSUnitElectricCurrent"),
+        (UnitElectricPotentialDifference.self, "NSUnitElectricPotentialDifference"),
+        (UnitElectricResistance.self, "NSUnitElectricResistance"),
+        (UnitEnergy.self, "NSUnitEnergy"),
+        (UnitFrequency.self, "NSUnitFrequency"),
+        (UnitFuelEfficiency.self, "NSUnitFuelEfficiency"),
+        (UnitIlluminance.self, "NSUnitIlluminance"),
+        (UnitLength.self, "NSUnitLength"),
+        (UnitMass.self, "NSUnitMass"),
+        (UnitPower.self, "NSUnitPower"),
+        (UnitPressure.self, "NSUnitPressure"),
+        (UnitSpeed.self, "NSUnitSpeed"),
+        (UnitVolume.self, "NSUnitVolume"),
+        (UnitTemperature.self, "NSUnitTemperature"),
+    ]
+#if !(os(iOS) || os(Android)
```

Comment:
Missing trailing `)`

Reply:
Clearly pulling changes between two repositories is a great idea 🙄